### PR TITLE
Adjusted Kuryr documentation

### PR DESCRIPTION
**Pull request by tzumainn**
A few recommended Kuryr settings have changed; this PR reflects those changes.

### CODE REVIEW

File: install_config/configuring_kuryrsdn.adoc [hunk 1]
```diff
@@ -69,10 +69,9 @@ Kuryr-Kubernetes as the network plug-in:
  # Set userspace so that there are no iptables remains.
  openshift_node_proxy_mode='userspace'
 
- # Disable management of the OpenShift Registry and OpenShift Router. If you
- # need those services, install them manually after OpenShift deployment using
- # `oadm`. This way services will get correct IPs provided by Kuryr.
- openshift_hosted_manage_registry=false
+ # Disable management of the OpenShift Router. If you need this service, install
+ # it manually after OpenShift deployment using  `oadm`. This way services
```
Comment (ahardin-rh) [inline install_config/configuring_kuryrsdn.adoc]: This ensures services will have correct IPs provided by Kuryr.

File: install_config/configuring_kuryrsdn.adoc [hunk 2]
```diff
@@ -109,6 +108,9 @@ Kuryr-Kubernetes as the network plug-in:
  kuryr_openstack_service_subnet_id=3b31a106-4084-4db9-bc0c-00b97afe186e
 ----
 
+You'll also need to specify an OpenStack cloud provider as described in the
```
Comment (ahardin-rh) [inline install_config/configuring_kuryrsdn.adoc]: You must also specify an OpenStack cloud provider

File: install_config/configuring_kuryrsdn.adoc [hunk 3]
```diff
@@ -109,6 +108,9 @@ Kuryr-Kubernetes as the network plug-in:
...
+xref:configuring_openstack.adoc#openstack-configuring-masters-ansible[OpenStack configuration documentation].
```
Comment (ahardin-rh) [inline install_config/configuring_kuryrsdn.adoc]: I think that the link needs to be formatted as xref:../

Please check that the link works in a preview build.

**Comment by ahardin-rh** [comment]
Thanks for this! I added a few comments. What versions of our documentation is this targeting? Is it for 3.10+?

File: install_config/configuring_kuryrsdn.adoc (hunk 3 above)
Comment (tzumainn) [reply install_config/configuring_kuryrsdn.adoc]: Hi! I *think* this works; it worked for me in a local checkout.

**Comment by tzumainn** [comment]
@ahardin-rh Thanks for the review! I've updated based on your suggestions. These changes would apply for 3.10 and forward.

@luis5tb Could you verify that my changes are correct from a technical perspective?

File: install_config/configuring_kuryrsdn.adoc (hunk 3 above)
Comment (ahardin-rh) [reply install_config/configuring_kuryrsdn.adoc]: @tzumainn Okay, thanks! Also, we'll need to squash the 2 commits into 1 commit per our publishing process. Let me know if you need help with that.

Comment (tzumainn) [reply install_config/configuring_kuryrsdn.adoc]: Ah, wasn't sure what the standard was. I've squashed the commits, thanks!

File: install_config/configuring_kuryrsdn.adoc [hunk 4]
```diff
@@ -69,10 +69,9 @@ Kuryr-Kubernetes as the network plug-in:
  # Set userspace so that there are no iptables remains.
  openshift_node_proxy_mode='userspace'
 
- # Disable management of the OpenShift Registry and OpenShift Router. If you
- # need those services, install them manually after OpenShift deployment using
- # `oadm`. This way services will get correct IPs provided by Kuryr.
- openshift_hosted_manage_registry=false
+ # Disable management of the OpenShift Router. If you need this service, install
```
Comment (luis5tb) [inline install_config/configuring_kuryrsdn.adoc]: why you need to disable router? I believe there is no need to disable registry, nor router right now

Comment (tzumainn) [reply install_config/configuring_kuryrsdn.adoc]: Ah, okay! I wasn't sure. I'll update.

**Comment by tzumainn** [comment]
@luis5tb I've removed the bit you pointed out, thanks!

**Comment by tzumainn** [comment]
@ahardin-rh hi! is there anything else that needs to be done to have this merged?

**Comment by ahardin-rh** [comment]
@tzumainn Just cross-checked that link and it works. Merging now. Sorry for the delay!

**Comment by tzumainn** [comment]
@ahardin-rh no problem, thanks a lot!

**Pull request merged**